docs/sources/_index.md.t

```diff
@@ -101,8 +101,6 @@ one minor release is moved.
 
 Patch and security releases may be created at any time.
 
-[Milestones]: https://github.com/grafana/agent/milestones
```

Comment:
This was a stale link; it was never actually referenced anywhere so this is a no-op change to docs.